microprot/snakemake/Snakefile

```diff
@@ -23,6 +23,11 @@ try:
 except KeyError:
     pass
 
+# create output directories if they don't exist
+for _out_dir_name in ['PDB', 'CM', 'AB', 'log', 'pkg']:
+    _out_dir_path = '%s/%s' % (config['MICROPROT_OUT'], _out_dir_name)
+    if not exists(_out_dir_path):
```

Comment:
I'm not sure if I want to do that. It does the check for each snakemake run, so if I already have 1,000 files there I don't want to create those folders anew.